### PR TITLE
[#2] [Chore] Setup Fastlane swift for the project

### LINKED ISSUE #2: [Chore] Setup Fastlane swift for the project
## Why

- We need to set up `Fastlane swift` for the project so developers can set up CI/ CD based on it.

## Acceptance Criteria

- We can run a local lane.

### PR DESCRIPTION
**Pull request by Thieurom**
- Close #2

## What happened 👀

We need to set up Fastlane swift for the project so developers can set up CI/ CD based on it.

## Insight 📝

- Install `fastlane` as a Ruby gem
- Setup with `fastlane init swift`
- Add an example lane (for testing locally purpose)
- Update `gitignore` with `fastlane`'s stuff

## Proof Of Work 📹

- Run the example lane successfully on the local machine:
Screenshot 2022-11-29 at 10 30 54
fastlane

### CODE REVIEW

**Review by minhnimble — APPROVED** [review]
The rest LGTM, nice work 🚀

File: fastlane/Appfile.swift [hunk 1]
```diff
@@ -0,0 +1,7 @@
+var appIdentifier: String { return "[[APP_IDENTIFIER]]" } // The bundle identifier of your app
+var appleID: String { return "[[APPLE_ID]]" } // Your Apple Developer Portal username
+
+
+
+// For more information about the Appfile, see:
+//     https://docs.fastlane.tools/advanced/#appfile
```
[the remark this answers is not in the document]
Comment (Thieurom) [reply fastlane/Appfile.swift]: @minhnimble
In the `fastlane` directory we have `Appfile.swift` and `Fastfile.swift` , we will add appIds and custom lanes on these files. On the `fastlane/swift` directory we have two files with the same name but these are automatically generated by `fastlane` and we won't touch them 🙏

Besides that, the `fastlane/Appfile.swift` has been removed redundant spacings in this [update](https://github.com/nimblehq/nimble-crypto-ios/pull/20/commits/baaa1a96a907ec873ba3b6ed32d28518b83ec973)

[the remark this answers is not in the document]
Comment (minhnimble) [reply fastlane/Appfile.swift]: oh thanks for the insights on the generated file, this is new knowledge to me 🙏

**Review by nkhanh44 — APPROVED** [review]
All good to me 😄 .

**Pull request merged**